contracts/Baal.sol

```diff
@@ -483,6 +482,7 @@ contract Baal is CloneFactory, Module {
                 keccak256(bytes(name)),
                 block.chainid,
                 address(this)
+                // TODO dont we need version in this Domain Seperator?
```

Comment:
left this comment here in regards to 

dont we need version in domain seperator?
https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator
```
Definition of domainSeparator
domainSeparator = hashStruct(eip712Domain)
where the type of eip712Domain is a struct named EIP712Domain with one or more of the below fields. Protocol designers only need to include the fields that make sense for their signing domain. Unused fields are left out of the struct type.

string name the user readable name of signing domain, i.e. the name of the DApp or the protocol.
string version the current major version of the signing domain. Signatures from different versions are not compatible.
uint256 chainId the EIP-155 chain id. The user-agent should refuse signing if it does not match the currently active chain.
address verifyingContract the address of the contract that will verify the signature. The user-agent may do contract specific phishing prevention.
bytes32 salt an disambiguating salt for the protocol. This can be used as a domain separator of last resort.